app/main/views.py

```diff
@@ -1,31 +1,27 @@
-from flask import render_template, request
+from flask import render_template
 from . import main
 from ..models import Resource
 import json
 
 
-@main.route('/', methods=['GET'])
+@main.route('/')
 def index():
     return render_template('main/index.html')
 
 
-@main.route('/get-resource', methods=['GET'])
-def get_resource():
-    Resource.generate_fake()
-    names = Resource.query.with_entities(Resource.name)
-    lats = Resource.query.with_entities(Resource.latitude)
-    longs = Resource.query.with_entities(Resource.longitude)
-    data = []
-    counter = 0
-    for name in names:
-        this_pin = {'Name': name, 'Latitude': lats[counter], 'Longitude':
-            longs[counter]}
-        counter = counter + 1
-        data.append(this_pin)
-    return json.dumps(data)
+@main.route('/get-resources')
+def get_resources():
+    resources = Resource.query.all()
+    resources_as_dicts = [resource.__dict__ for resource in resources]
+    # .__dict__ returns the SQLAlchemy object as a dict, but it also adds a
+    # field '_sa_instance_state' that we don't need, so we delete it.
+    for dict in resources_as_dicts:
```

Comment:
`dict` is a data type, so for safety and clarify we probably shouldn't use it as a variable type. Perhaps just `d` instead?